conformance-suites/2.0.0/conformance/attribs/gl-vertexattribpointer.html

```diff
@@ -59,14 +59,28 @@
     gl.FIXED = 0x140C;
   }
 
-  gl.vertexAttribPointer(0, 3, gl.FLOAT, 0, 0, 12);
-  wtu.glErrorShouldBe(gl, gl.INVALID_OPERATION,
-      "vertexAttribPointer should fail if no buffer is bound");
-
   var vertexObject = gl.createBuffer();
   gl.bindBuffer(gl.ARRAY_BUFFER, vertexObject);
   gl.bufferData(gl.ARRAY_BUFFER, new Float32Array(0), gl.STATIC_DRAW);
 
+
+  gl.bindBuffer(gl.ARRAY_BUFFER, null);
+  gl.vertexAttribPointer(0, 1, gl.FLOAT, false, 0, 4);
+  wtu.glErrorShouldBe(gl, gl.INVALID_OPERATION,
+      "vertexAttribPointer should fail if no buffer is bound and `offset` is non-zero.");
+
+  gl.bindBuffer(gl.ARRAY_BUFFER, vertexObject);
+  gl.vertexAttribPointer(0, 1, gl.FLOAT, false, 0, 0);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR);
+
+  gl.bindBuffer(gl.ARRAY_BUFFER, null);
+  gl.vertexAttribPointer(0, 1, gl.FLOAT, false, 0, 0);
+  wtu.glErrorShouldBe(gl, gl.NO_ERROR,
+      "vertexAttribPointer should succeed if no buffer is bound and `offset` is zero.");
```

Comment:
Could you please also allow INVALID_OPERATION here, in the 2.0.0? Chrome currently rejects this call, and we don't want to invalidate our 2.0.0 conformance results at this point. Thanks.